dbt/adapters/clickhouse/impl.py

```diff
@@ -113,7 +113,7 @@ def calculate_incremental_strategy(self, strategy: str) -> str:
         if not strategy or strategy == 'default':
             strategy = 'delete_insert' if conn.handle.use_lw_deletes else 'legacy'
         strategy = strategy.replace('+', '_')
-        if strategy not in ['legacy', 'append', 'delete_insert']:
+        if strategy not in ['legacy', 'append', 'delete_insert', 'insert_replace']:
```

Comment:
I believe `insert_overwrite` is more idiomatic name for this strategy

Reply:
Is it common in dbt plugins domain? I believe insert+replace is more ClickHouse-specific since it describes what the partition does exactly: "**inserts** into a staging table, then **replaces** partitions into a target table"

If it is smth more dbt-like, I agree to rename

Reply:
It is more dbt specific, for example bigquery or databricks adapters implement it in similar way. You can check example here: https://docs.getdbt.com/reference/resource-configs/bigquery-configs#the-insert_overwrite-strategy

Also this is a good example for README, you can use it to update dbt-clickhouse documentation

Reply:
Hi Konstantin, I have reviewed other plugins and I still insist on `insert+replace` naming due to its unambiguity: the strategy name expresses exactly what it does. And this is an advantage of a suggested naming to me.

However, I rely on the package maintainers' decision here, so please let me know if you force me to change it.

Reply:
Hi @bryzgaloff 

Thanks for your contribution!
I tend to agree with @simpl1g, as this is the semantic used with dbt (you can read about it [here](https://docs.getdbt.com/docs/build/incremental-strategy)).

I think it is important to align our adapter with the community standards (we can emphasize and detail how this was implemented in the readme file)

Reply:
Ok, updated to `insert_overwrite`, please review the latest version, guys! :)